parsers/ocaml.c

```diff
@@ -119,48 +123,50 @@ typedef struct sOcaKeywordDesc {
-	{ "match"     , OcaKEYWORD_match     }, 
-	{ "method"    , OcaKEYWORD_method    }, 
-	{ "module"    , OcaKEYWORD_module    }, 
-	{ "mutable"   , OcaKEYWORD_mutable   }, 
-	{ "object"    , OcaKEYWORD_object    }, 
-	{ "of"        , OcaKEYWORD_of        }, 
-	{ "rec"       , OcaKEYWORD_rec       }, 
-	{ "sig"       , OcaKEYWORD_sig       }, 
-	{ "struct"    , OcaKEYWORD_struct    }, 
-	{ "then"      , OcaKEYWORD_then      }, 
-	{ "try"       , OcaKEYWORD_try       }, 
-	{ "type"      , OcaKEYWORD_type      }, 
-	{ "val"       , OcaKEYWORD_val       }, 
+	{ "and"       , OcaKEYWORD_and       },
+	{ "begin"     , OcaKEYWORD_begin     },
+	{ "class"     , OcaKEYWORD_class     },
+	{ "do"        , OcaKEYWORD_do        },
+	{ "done"      , OcaKEYWORD_done      },
+	{ "else"      , OcaKEYWORD_else      },
+	{ "end"       , OcaKEYWORD_end       },
+	{ "exception" , OcaKEYWORD_exception },
+	{ "for"       , OcaKEYWORD_for       },
+	{ "fun"       , OcaKEYWORD_fun       },
+	{ "function"  , OcaKEYWORD_fun       },
+	{ "functor"   , OcaKEYWORD_functor   },
+	{ "if"        , OcaKEYWORD_if        },
+	{ "in"        , OcaKEYWORD_in        },
+	{ "let"       , OcaKEYWORD_let       },
+	{ "match"     , OcaKEYWORD_match     },
+	{ "method"    , OcaKEYWORD_method    },
+	{ "module"    , OcaKEYWORD_module    },
+	{ "mutable"   , OcaKEYWORD_mutable   },
+	{ "object"    , OcaKEYWORD_object    },
+	{ "of"        , OcaKEYWORD_of        },
+	{ "rec"       , OcaKEYWORD_rec       },
+	{ "sig"       , OcaKEYWORD_sig       },
+	{ "struct"    , OcaKEYWORD_struct    },
+	{ "then"      , OcaKEYWORD_then      },
+	{ "try"       , OcaKEYWORD_try       },
+	{ "type"      , OcaKEYWORD_type      },
+	{ "val"       , OcaKEYWORD_val       },
 	{ "value"     , OcaKEYWORD_value     }, /* just to handle revised syntax */
-	{ "virtual"   , OcaKEYWORD_virtual   }, 
-	{ "while"     , OcaKEYWORD_while     }, 
-	{ "with"      , OcaKEYWORD_with      }, 
-
-	{ "or"        , Tok_Op               }, 
-	{ "mod "      , Tok_Op               }, 
-	{ "land "     , Tok_Op               }, 
-	{ "lor "      , Tok_Op               }, 
-	{ "lxor "     , Tok_Op               }, 
-	{ "lsl "      , Tok_Op               }, 
-	{ "lsr "      , Tok_Op               }, 
-	{ "asr"       , Tok_Op               }, 
-	{ "->"        , Tok_To               }, 
-	{ "true"      , Tok_Val              }, 
+	{ "virtual"   , OcaKEYWORD_virtual   },
+	{ "while"     , OcaKEYWORD_while     },
+	{ "with"      , OcaKEYWORD_with      },
+
+	{ "or"        , Tok_Op               },
+	{ "mod "      , Tok_Op               },
+	{ "land "     , Tok_Op               },
+	{ "lor "      , Tok_Op               },
+	{ "lxor "     , Tok_Op               },
+	{ "lsl "      , Tok_Op               },
+	{ "lsr "      , Tok_Op               },
+	{ "asr"       , Tok_Op               },
+	{ "->"        , Tok_To               },
+	{ ":"         , Tok_Of               },
+	{ "true"      , Tok_Val              },
```

Comment:
I totally agree, especially as it also did hide a bug: `OcaKEYWORD_function` is *never* mapped here, so no token using it are ever created, and then all code paths using it area effectively dead.  Bot "fun" and "function" are mapped to `OcaKEYWORD_fun`.

Also, if ignoring whitespace changes, we see what actually was added: `"if", OcaKEYWORD_if` and `":", Tok_Of`